scripts/singularity_cmd

```diff
@@ -78,6 +78,10 @@ if [ -n "${DATALAD_CONTAINER_NAME:-}" ]; then
 	export APPTAINERENV_DATALAD_CONTAINER_NAME="$DATALAD_CONTAINER_NAME"
 fi
 
+# fix non-writable matplotlib cache dir in apptainer https://github.com/ReproNim/containers/issues/97                                  
+export SINGULARITYENV_MPLCONFIGDIR=/tmp                                                                                                
+export APPTAINERENV_MPLCONFIGDIR=/tmp         
```

Comment:
would it create that path if doesn't exist?  if so, I would have preferred to have some dedicated folder there e.g.

```suggestion
export SINGULARITYENV_MPLCONFIGDIR=/tmp/mpl-config
export APPTAINERENV_MPLCONFIGDIR=/tmp/mpl-config
```

note: bunch of trailing spaces removal -- check/fix other lines too please